### ISSUE TITLE
Unable to save Text via plugin_ckeditor

### ISSUE THREAD

**Issue by abchk1234** [qcubed/plugin_ckeditor#6]
Hi,

I added plugin_ckeditor as specified in the README, and I create a ckeditor instance like:

```
$this->txtBody = new \QCubed\Plugin\QCKEditor($this);
```

But when I try to get the content posted in the ckeditor like:

```
$strContent = $this->txtBody->Text;
```

I get nothing, using debugging shows it as having null value..

```
$this->txtBody = {QCubed\Plugin\QCKEditor} [72]
 strJsReadyFunc = "function(){}"
 intColumns = 0
 strText = null
 strLabelForRequired = "%s is required"
 strLabelForRequiredUnnamed = "Required"
 strLabelForTooShort = "%s must have at least %s characters"
 strLabelForTooShortUnnamed = "Must have at least %s characters"
 strLabelForTooLong = "%s must have at most %s characters"
 strLabelForTooLongUnnamed = "Must have at most %s characters"
 strPlaceholder = ""
 strFormat = "%s"
 intMaxLength = 0
 intMinLength = 0
 intRows = 0
 strTextMode = "MultiLine"
 strCrossScripting = "Allow"
 objHTMLPurifierConfig = null
 blnAutoTrim = false
 intSanitizeFilter = null
 mixSanitizeFilterOptions = null
 intValidateFilter = null
 mixValidateFilterOptions = null
 strLabelForInvalid = null
 strTestingId = null
 objWrapperStyler = {QTagStyler} [2]
 mixCausesValidation = false
 blnRequired = false
 strValidationError = null
 blnVisible = true
 blnDisplay = true
 strPreferredRenderMethod = "Render"
 strHtmlBefore = null
 strHtmlAfter = null
 strInstructions = null
 strWarning = null
 objDraggable = null
 objResizable = null
 objDroppable = null
 strControlId = "c5"
 objForm = {Index} [44]
 objParentControl = {HzFcBlogEditPostPage} [77]
 objChildControlArray = {array} [0]
 strName = null
 blnRendered = false
 blnRendering = false
 blnOnPage = true
 blnModified = false
 blnWrapperModified = false
 strRenderMethod = "Render"
 strCustomAttributeArray = null
 strCustomStyleArray = null
 objActionArray = {array} [0]
 mixActionParameter = null
 blnUseWrapper = true
 strAttributeScripts = null
 strCssClass = null
 blnMinimize = false
 strJavaScripts = "/vendor/twbs/bootstrap/dist/js/bootstrap.min.js,/vendor/qcubed/plugin/bootstrap/js/qshim.js,/vendor/qcubed/plugin/ckeditor/js/QCKSetup.js,/vendor/ckeditor/ckeditor/ckeditor.js,/vendor/ckeditor/ckeditor/adapters/jquery.js"
 strStyleSheets = null
 strFormAttributes = null
 blnActionsMustTerminate = false
 blnScriptsOnly = false
 blnIsBlockElement = false
 objWatcher = null
 objLinkedNode = null
 blnWrapLabel = false
 blnSaveState = false
 attributes = {array} [0]
 styles = {array} [0]
 strValidationState = null
 objLabelStyler = null
 strHorizontalClass = null
```

**Comment by spekary** [comment]
I did a little digging. You will need a server post for now for the info to get from the javascript widget to the PHP. Not sure yet if there is an ajax way to do it yet.

**Comment by abchk1234** [comment]
Using QServerControlAction instead of QAjaxControlAction worked :-)

Feel free to close this issue depending on whether this is a bug or not (probably a note in the README stating that it only works with Server action would be appropriate).

**Comment by spekary** [comment]
The issue is that we need a mechanism for custom controls to post values through both ajax and server calls. Working on it now.

**Comment by spekary** [comment]
Fixed in #7
